servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -1405,6 +1457,63 @@ private static ClassName routeInterfaceClass(final MethodDescriptorProto methodP
                         : blocking ? BlockingRoute : Route);
     }
 
+    private static CodeBlock routeDefinition(final RpcInterface rpcInterface, final String routeName,
+                                             final ClassName inClass, final ClassName outClass) {
+        final ClassName routeInterfaceClass = routeInterfaceClass(rpcInterface.methodProto, rpcInterface.blocking);
+        final TypeName routeType = ParameterizedTypeName.get(routeInterfaceClass, inClass, outClass);
+        if (rpcInterface.blocking && rpcInterface.methodProto.getServerStreaming()) {
+            return CodeBlock.builder()
```

Comment:
Could a common base class be used? This ends up being boiler plate repeated for each RPC

Reply:
Not each, only for blocking with streaming response as a temporary need due to 2 methods on the interface. It will be simplified as soon as we start removing deprecations.

Reply:
There is not much benefit of moving it to a separate class instead of an anonymous class because you need a different one for each RPC. Since it's generated code that is required temporarily, it's ok to have it as in-place anonymous class.